alws/crud/build_node.py

```diff
@@ -58,7 +59,7 @@ async def get_available_build_task(
                         models.Build.platform_flavors).selectinload(
                         models.PlatformFlavour.repos),
                     selectinload(models.BuildTask.rpm_module)
-            ).order_by(models.BuildTask.id)
+            ).order_by(models.BuildTask.id.asc())
```

Comment:
It's necessary? because I'm think that if will affect us in future

Reply:
Should not affect, because this only ensures we will pick up the task with smallest ID